web/src/features/map/map-layers/StatesLayer.tsx

```diff
@@ -1,42 +1,32 @@
 import { useTheme } from 'hooks/theme';
-import { Layer, Source } from 'react-map-gl';
+import { Layer, Source } from 'react-map-gl/maplibre';
 
 import { useGetGeometries } from '../map-utils/getMapGrid';
 
 export default function StatesLayer() {
   const { statesGeometries } = useGetGeometries();
   const theme = useTheme();
 
-  const statesBorderStyle = {
-    'line-color': theme.stateBorderColor,
-    'line-width': 1.4,
-    'line-dasharray': [1, 1],
-    'line-opacity': ['interpolate', ['linear'], ['zoom'], 2.8, 0, 3, 0.9],
-  } as mapboxgl.LinePaint;
-
-  const stateLabelLayout = {
-    'symbol-placement': 'point',
-    'text-size': 12,
-    'text-letter-spacing': 0.12,
-    'text-transform': 'uppercase',
-    'text-font': ['poppins-semibold'],
-  } as mapboxgl.SymbolLayout;
-
```

Comment:
Honestly I have no idea why I had to inline these for the typing to work but that is the only thing that seems to fix the issue.

If someone can explain it please do 😅